run

```diff
@@ -9,54 +9,52 @@ mkdir -p $HAPROXY_SERVICE/env
 if [ -n "${PORTS-}" ]; then
   echo $PORTS > $HAPROXY_SERVICE/env/PORTS
 else
-  echo "Define $PORTS with a comma-separated list of ports to which HAProxy binds" >&2
+  echo 'Define $PORTS with a comma-separated list of ports to which HAProxy binds' >&2
   exit 1
 fi
 
-# Iterate through the arguments to see if any SSL certificate is provided.
-prev=0
-for key in "$@"
-do
-
-case $prev in
-    --ssl-certs)
-      SSL_CERTS=$key
-    ;;
-    *)
-            # unknown option
-    ;;
-esac
-prev=$key
+# Find the --ssl-certs arg if one was provided,
+# get the certs and remove them and the arg from the list
+# of positional parameters so we don't duplicate them
+# further down when we pass $@ to marathon_lb.py
+declare -i ssl_certs_pos=0
+for ((i=1; i<=$#; i++)); do
+  if [ ${!i} = '--ssl-certs' ]; then
+    ssl_certs_pos=$(($i+1))
+    break
```

Comment:
FYI I just added this `break` and squashed the commits. Other than that it's the same code you already tested.